<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EELEditor</class>
 <widget class="QMainWindow" name="EELEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEL Scripting IDE</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>19</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose_filw"/>
    <addaction name="separator"/>
    <addaction name="actionRun_code"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
    <addaction name="actionGo_to_line"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionJump_to_function"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_init"/>
    <addaction name="actionGo_to_sample"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionEEL2_documentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen_file"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionFind_Replace"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionRun_code"/>
   <addaction name="actionFreeze"/>
  </widget>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/open_folder.svg</normaloff>:/icons/open_folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_filw">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>Close file</string>
   </property>
   <property name="toolTip">
    <string>Close file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/saveas.svg</normaloff>:/icons/saveas.svg</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/undo.svg</normaloff>:/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/redo.svg</normaloff>:/icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/FindInFile_16x.svg</normaloff>:/icons/FindInFile_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Find/Replace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGo_to_line">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/GoToCurrentLine_16x.svg</normaloff>:/icons/GoToCurrentLine_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Go to line...</string>
   </property>
   <property name="toolTip">
    <string>Go to line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionJump_to_function">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/GoToMethod_16x.svg</normaloff>:/icons/GoToMethod_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Jump to function...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionGo_to_init">
   <property name="text">
    <string>Go to @init</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionGo_to_sample">
   <property name="text">
    <string>Go to @sample</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAutomatic_Parentheses">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic Parentheses</string>
   </property>
  </action>
  <action name="actionReplace_Tabs_with_Spaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replace Tabs with Spaces</string>
   </property>
  </action>
  <action name="actionEEL2_documentation">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/help.svg</normaloff>:/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>EEL2 documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRun_code">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/StatusRun_16x.svg</normaloff>:/icons/StatusRun_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Run code</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/NewFile_16x.svg</normaloff>:/icons/NewFile_16x.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New script wizard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFreeze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="editorresources.qrc">
     <normaloff>:/icons/RunPause_16x.svg</normaloff>:/icons/RunPause_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Freeze</string>
   </property>
   <property name="toolTip">
    <string>Freeze</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="editorresources.qrc"/>
 </resources>
 <connections/>
</ui>
